{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1",
    "cveMetadata": {
        "cveId": "CVE-2025-0290",
        "assignerOrgId": "ceab7361-8a18-47b1-92ba-4d7d25f6715a",
        "state": "PUBLISHED",
        "assignerShortName": "GitLab",
        "dateReserved": "2025-01-06T22:30:48.611Z",
        "datePublished": "2025-01-28T08:45:09.560Z",
        "dateUpdated": "2025-01-28T14:41:52.667Z"
    },
    "containers": {
        "cna": {
            "title": "Loop with Unreachable Exit Condition ('Infinite Loop') in GitLab",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "An issue has been discovered in GitLab CE/EE affecting all versions starting from 15.0 prior to 17.5.5, from 17.6 prior to 17.6.3, and from 17.7 prior to 17.7.1. Under certain conditions, processing of CI artifacts metadata could cause background jobs to become unresponsive."
                }
            ],
            "affected": [
                {
                    "vendor": "GitLab",
                    "product": "GitLab",
                    "repo": "git://git@gitlab.com:gitlab-org/gitlab.git",
                    "cpes": [
                        "cpe:2.3:a:gitlab:gitlab:*:*:*:*:*:*:*:*"
                    ],
                    "versions": [
                        {
                            "version": "15.0",
                            "status": "affected",
                            "lessThan": "17.6.4",
                            "versionType": "semver"
                        },
                        {
                            "version": "17.7",
                            "status": "affected",
                            "lessThan": "17.7.2",
                            "versionType": "semver"
                        },
                        {
                            "version": "17.8",
                            "status": "affected",
                            "lessThan": "17.8.0",
                            "versionType": "semver"
                        }
                    ],
                    "defaultStatus": "unaffected"
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "lang": "en",
                            "description": "CWE-835: Loop with Unreachable Exit Condition ('Infinite Loop')",
                            "cweId": "CWE-835",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "references": [
                {
                    "url": "https://gitlab.com/gitlab-org/gitlab/-/issues/372134",
                    "name": "GitLab Issue #372134",
                    "tags": [
                        "issue-tracking",
                        "permissions-required"
                    ]
                }
            ],
            "metrics": [
                {
                    "format": "CVSS",
                    "scenarios": [
                        {
                            "lang": "en",
                            "value": "GENERAL"
                        }
                    ],
                    "cvssV3_1": {
                        "version": "3.1",
                        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L",
                        "attackVector": "NETWORK",
                        "attackComplexity": "LOW",
                        "privilegesRequired": "LOW",
                        "userInteraction": "NONE",
                        "scope": "UNCHANGED",
                        "confidentialityImpact": "NONE",
                        "integrityImpact": "NONE",
                        "availabilityImpact": "LOW",
                        "baseScore": 4.3,
                        "baseSeverity": "MEDIUM"
                    }
                }
            ],
            "solutions": [
                {
                    "lang": "en",
                    "value": "Upgrade to versions 17.6.4, 17.7.2, 17.8.0 or above."
                }
            ],
            "credits": [
                {
                    "lang": "en",
                    "value": "This vulnerability was discovered internally by GitLab team member [Stan Hu](https://gitlab.com/stanhu).",
                    "type": "finder"
                }
            ],
            "providerMetadata": {
                "orgId": "ceab7361-8a18-47b1-92ba-4d7d25f6715a",
                "shortName": "GitLab",
                "dateUpdated": "2025-01-28T08:45:09.560Z"
            }
        },
        "adp": [
            {
                "metrics": [
                    {
                        "other": {
                            "type": "ssvc",
                            "content": {
                                "timestamp": "2025-01-28T14:41:43.330404Z",
                                "id": "CVE-2025-0290",
                                "options": [
                                    {
                                        "Exploitation": "none"
                                    },
                                    {
                                        "Automatable": "no"
                                    },
                                    {
                                        "Technical Impact": "partial"
                                    }
                                ],
                                "role": "CISA Coordinator",
                                "version": "2.0.3"
                            }
                        }
                    }
                ],
                "title": "CISA ADP Vulnrichment",
                "providerMetadata": {
                    "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
                    "shortName": "CISA-ADP",
                    "dateUpdated": "2025-01-28T14:41:52.667Z"
                }
            }
        ]
    }
}